Makefile

```diff
@@ -12,23 +12,11 @@ test-unit: tidy
 
 .PHONY: test-integration
 test-integration: tidy
-	@go test -v ./test/integration/...
+	@go test -v ./test/integration/... -count=1
```

Comment:
I'd noticed this `-count=1` and thought nothing of it at the time. Now that you've added it here I finally bothered to look it up and was surprised to learn that this disables caching---what a weird way for go to specify "no caching".... Not only that but any non-cacheable argument disables caching. Why not a simple `--no-caching` flag? 
> To disable test caching, use any test flag or argument other than the cacheable flags. The idiomatic way to disable test caching explicitly is to use -count=1.

Either way, behavior of the tool and its arguments is not up to you, but I think its worth adding a comment documenting the fact that we use `-count=1` to disable caching and not because we only want to run the test once. Also, since it looks like we want this flag in all of our tests, we could define a variable for go test or go test flags. In fact, we do this in zpr:
```make
# The idiomatic way to disable test caching explicitly is to use -count=1
GOTEST=go test -count=1
#...
test-integration: tidy
    @$(GOTEST) -v ./test/integration/...
```

Reply:
Yes, that is a very strange syntax for this options. I'll take your suggestion to do it the clean way like in ZPR.